internal/pkg/manifest/svc.go

```diff
@@ -204,11 +257,11 @@ func (a *AdvancedCount) hasAutoscaling() bool {
 func (a *AdvancedCount) validScalingFields() []string {
 	switch a.workloadType {
 	case LoadBalancedWebServiceType:
-		return []string{"cpu_percentage", "memory_percentage", "requests", "response_time"}
+		return []string{"cpu_percentage", "memory_percentage", "requests", "response_time", "cooldown"}
 	case BackendServiceType:
-		return []string{"cpu_percentage", "memory_percentage"}
+		return []string{"cpu_percentage", "memory_percentage", "cooldown"}
 	case WorkerServiceType:
-		return []string{"cpu_percentage", "memory_percentage", "queue_delay"}
+		return []string{"cpu_percentage", "memory_percentage", "queue_delay", "cooldown"}
```

Comment:
I don't think we should add `cooldown` here. Same for `hasScalingFieldsSet`. Reason is `cooldown` does not enable any scaling policy and users are not supposed to just set `cooldown` without setting any auto scaling policy. Instead, in `validate.go` we probably want to make sure users set at least one of the `validScalingFields` if they set `cooldown`.